setupPRarea

```diff
@@ -0,0 +1,24 @@
+#!/bin/tcsh
+
+setenv SCRAM_ARCH slc6_amd64_gcc493
+setenv PATH /afs/cern.ch/cms/common/:$PATH
+
+#echo Renew proxy
+#voms-proxy-init -valid 100:00 -voms cms
+
+setenv pr `pwd | sed 's/.*pr//'`
```

Comment:
ill remove setupPR, it was a convenient script for quick setup that shouldnt be added